subprojects/base-services/src/main/java/org/gradle/internal/progress/BuildOperationState.java

```diff
@@ -17,7 +17,7 @@
 package org.gradle.internal.progress;
 
 public interface BuildOperationState {
-    Object getId();
+    OperationIdentifier getId();
```

Comment:
@jjohannes nevermind, I'm backing out this change for now because it is not strictly necessary and I found a few more places I'd have to use this type.